wandb/sdk/data_types/converters.py

```diff
@@ -0,0 +1,158 @@
+import logging
+import re
+from typing import cast, Optional, Sequence, TYPE_CHECKING, Union
+
+from six.moves.collections_abc import Sequence as SixSequence
```

Comment:
same as before replace this as we don't need to support py27 anymore

Reply:
fixed in next commit